library/reminders.php

```diff
@@ -227,15 +228,15 @@ function update_reminders($dateTarget = '', $patient_id = '', $start = null, $ba
         // as described above, need to pass in each patient_id
         // Collect all patient ids
         $patientData = buildPatientArray('', '', '', $start, $batchSize);
-        for ($iter = 0; $row = sqlFetchArray($rez); $iter++) {
+        // comment below as doesn't make sense. where's $rez? sjp 05/27/20
+        /*for ($iter = 0; $row = sqlFetchArray($rez); $iter++) {
             $patientData[$iter] = $row;
-        }
+        }*/
```

Comment:
yep, that unused for block needs to go. this probably fell out when updated the rule engine to support batches (this was added for when running thousands of patients in a reports).